roles/common/defaults/main.yml

```diff
@@ -233,6 +233,7 @@ gi_patches:
   - { category: "PSU", base: "12.1.0.2.0", release: "12.1.0.2.200414", patchnum: "30783882", patchfile: "p30783882_121020_Linux-x86-64.zip", patch_subdir: "/30805421", prereq_check: FALSE, method: "opatchauto apply", ocm: FALSE, upgrade: FALSE }
   - { category: "PSU", base: "12.1.0.2.0", release: "12.1.0.2.200714", patchnum: "31326400", patchfile: "p31326400_121020_Linux-x86-64.zip", patch_subdir: "/31305174", prereq_check: FALSE, method: "opatchauto apply", ocm: FALSE, upgrade: FALSE }
   - { category: "PSU", base: "12.1.0.2.0", release: "12.1.0.2.201020", patchnum: "31720761", patchfile: "p31720761_121020_Linux-x86-64.zip", patch_subdir: "/31718737", prereq_check: FALSE, method: "opatchauto apply", ocm: FALSE, upgrade: FALSE }
+# Not including JAN/APR 2021 RUs for 12.1.0.2, since some customers, when they try to download 2021 RUs, may get: You do not have privileges to download Software Extended Support patches.
```

Comment:
12.1 has been in an extended support phase for quite a while;  in order to provide a secure-by-default configuration, we generally include patches that require extended support.

Reply:
Ack, will include them

Reply:
Done